### PR TITLE
Lehao - phone number infinite bug fixes

### LINKED ISSUE #161: [PE-D] Phone Number infinite
As you can see here, the phone number can be extremely long. Users may, by accident, enter a very long phone number and it is not logical. 

Suggest to place some preventative measures to ensure that the phone number is not that long

But keep up the good work!

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: Timothyoung97/ped#5

### PR DESCRIPTION
**Pull request by Li-Lehao**
Fixes #161

### CODE REVIEW

File: src/main/java/seedu/placebook/model/person/Phone.java [hunk 1]
```diff
@@ -9,28 +9,30 @@
  */
 public class Phone {
 
+    public static final int MAXIMUM_PHONE_NUMBER_LENGTH = 20;
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
+            "Phone numbers should only contain numbers, it should be at least 3 digits long"
+                    + " and at most " + MAXIMUM_PHONE_NUMBER_LENGTH + " digits long";
     public static final String VALIDATION_REGEX = "\\d{3,}";
```
Comment (YaleChen299) [inline src/main/java/seedu/placebook/model/person/Phone.java:17]: I think changing this to `"\\d{3,20}";` will do the job

**Review by YaleChen299 — COMMENTED** [review]
LGTM just maybe one simpler implementation

File: src/main/java/seedu/placebook/model/person/Phone.java (hunk 1 above)
Comment (Li-Lehao) [reply src/main/java/seedu/placebook/model/person/Phone.java]: Oh ya changing the validation regex is a better way to do this. Thank you!

**Review by YaleChen299 — APPROVED** [review]
LGTM!

**Pull request merged**